extensions/kafka/tests/features/consumekafka.feature

```diff
@@ -75,18 +76,19 @@ Feature: Receiving data from using Kafka streaming platform using ConsumeKafka
       | Crime and Punishment | Фёдор Михайлович Достоевский  | Hex                    |
 
   Scenario Outline: ConsumeKafka transactional behaviour is supported
-    Given a ConsumeKafka processor set up in a "kafka-consumer-flow" flow
+    Given a Kafka server is set up
+    And ConsumeKafka processor is set up to communicate with that server
     And the "Topic Names" property of the ConsumeKafka processor is set to "ConsumeKafkaTest"
     And the "Honor Transactions" property of the ConsumeKafka processor is set to "<honor transactions>"
-    And a PutFile processor with the "Directory" property set to "/tmp/output" in the "kafka-consumer-flow" flow
+    And a PutFile processor with the "Directory" property set to "/tmp/output"
+    And PutFile is EVENT_DRIVEN
     And the "success" relationship of the ConsumeKafka processor is connected to the PutFile
-
-    And a kafka broker is set up in correspondence with the third-party kafka publisher
+    And PutFile's success relationship is auto-terminated
 
     When all instances start up
     And the publisher performs a <transaction type> transaction publishing to the "ConsumeKafkaTest" topic these messages: <messages sent>
 
-    Then <number of flowfiles expected> flowfiles are placed in the monitored directory in less than 15 seconds
+    Then there are <number of flowfiles expected> files in the "/tmp/output" directory in less than 15 seconds
```

Comment:
@fgerlits comment applies to this too: it's clearer to expect an event than a state, so the old version was better.